README.md

```diff
@@ -1,3 +1,3 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+1. In order to successfully work on this project, you have to create an environment variable with dotenv files (.env) and attach your database name to this. You should have two files, one for the test database and the other for the development database in order to interact with it as expected.
```

Comment:
You should explicitly state what needs to go in each .env file. You want to make it as clear and easy as possible for anyone who want's to explore your API. 

Ie add PGDATABASE=nc_news;
and PGDATABASE=nc_news_test;

It's really easy for the user to forget the semicolons here, or they might simply make a mistake. Make this change and merge your work, without submitting for another review 👍